source/mesh_deformation/fastscapecc.cc

```diff
@@ -0,0 +1,698 @@
+#include <iostream>
+
+#include <xtensor/xarray.hpp>
+#include <xtensor/xmath.hpp>
+#include <xtensor/xview.hpp>
+#include <xtensor/xadapt.hpp>
+
+#include <fastscapelib/flow/flow_graph.hpp>
+#include <fastscapelib/flow/sink_resolver.hpp>
+#include <fastscapelib/flow/flow_router.hpp>
+#include <fastscapelib/grid/raster_grid.hpp>
+#include <fastscapelib/eroders/diffusion_adi.hpp>
+#include <fastscapelib/eroders/spl.hpp>
+
+
+#include <aspect/global.h>
+
+#include <aspect/mesh_deformation/fastscapecc.h>
+#include <aspect/geometry_model/box.h>
+#include <deal.II/numerics/vector_tools.h>
+#include <aspect/postprocess/visualization.h>
+#include <ctime>
```

Comment:
Move this C++ header include down to where the other C++ headers are (specifically `<algorithm>`).